wiki/Tournaments/CWC/1/en.md

```diff
@@ -5,149 +5,149 @@ tags:
 
 # Catch the Beat World Cup
 
-The **Catch the Beat World Cup** (***CWC***) was a team-based osu!catch tournament hosted by the [osu! team](/wiki/People/The_Team). It was the first installment of the osu!catch World Cup.
+The **Catch the Beat World Cup** (***CWC***) was a single elimination team-based osu!catch tournament organized by various osu! community members. Despite not being hosted by (or being run under the direct provision of) the [osu!team](/wiki/People/The_Team), it was formally considered as the first installment of the osu!catch World Cup tournament series.
 
 ## Tournament schedule
 
 | Event | Timestamp |
 | --: | :-- |
 | Registration phase | 2011-09-30/2011-11-01 |
-| Round 1 | 2011-11-04/2011-11-06 |
-| Round 2 | 2011-11-11/2011-11-13 |
-| Round 3 | 2011-11-18/2011-11-20 |
-| Round 4 | 2011-11-25/2011-11-27 |
-| Round 5 | 2011-12-02/2011-12-04 |
-| Quarterfinals | 2012-01-07/2011-01-08 |
-| Semifinals | 2012-01-14/2011-01-15 |
-| Finals | 2012-02-05 |
+| Group Stage week 1 | 2011-11-02/2011-11-06 |
+| Group Stage week 2 | 2011-11-07/2011-11-13 |
+| Group Stage week 3 | 2011-11-14/2011-11-20 |
+| Group Stage week 4 | 2011-11-21/2011-11-27 |
+| Group Stage week 5 | 2011-11-28/2011-12-04 |
+| *Winter break* | 2011-12-05/2011-12-31 |
+| Quarterfinals | 2012-01-01/2011-01-08 |
+| Semifinals | 2012-01-09/2011-01-15 |
+| Finals | 2011-01-16/2012-02-05 |
```

Comment:
these seem like these should use an em dash instead of a slash?

Reply:
hmm i don't think so, i checked other world cup/major tournament articles and they all seem to use regular slash instead of em dash

![image](https://user-images.githubusercontent.com/36564236/106346194-d1195d80-62e7-11eb-8e30-73d2882b2380.png)